langchain/src/vectorstores/index.ts

```diff
@@ -8,3 +8,4 @@ export { PineconeStore } from "./pinecone.js";
 export { VectorStore, SaveableVectorStore } from "./base.js";
 export { SupabaseVectorStore } from "./supabase.js";
 export { PrismaVectorStore } from "./prisma.js";
+export { RedisVectorStore } from "./redis.js";
```

Comment:
This index file is deprecated, see: https://github.com/hwchase17/langchainjs/pull/685#discussion_r1166712827

Reply:
Fixed